.swiftlint.yml

```diff
@@ -44,6 +45,7 @@ opt_in_rules: # some rules are only opt-in
   - statement_position
   - explicit_init
   - shorthand_operator
+  - empty_count
```

Comment:
Okay we discussed it with the team, can you remove this rule from swiftlint?
And keep your code changes to `count => isEmpty`

We decided to not force this rule, but your code changes are nice to have regardless, sorry if this is confusing

Reply:
Awesome, thanks folks! I've pushed a commit disabling and removing the rule here: https://github.com/brave/brave-ios/pull/3381/commits/f860d83641384a3a5144b404964c4314a95bb0fe.

Reply:
looks good, thank you :) i will do final review tomorrow and we should be able to merge it